.vortex/cli/tests/Unit/TuiTest.php

```
@@ -173,13 +173,25 @@ public function testBox(
       Tui::box($content, $title);
     }
 
-    $actual = $output->fetch();
+    $actual = Strings::stripAnsiColors($output->fetch());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexCli\Utils\Strings' in method 'testBox'. (undefined)

(StaticAccess)